### PR TITLE
[i18n] Plural && Gender Support

### PR DESCRIPTION
**Pull request by delphiactual**
#1271

### CODE REVIEW

File: app/scripts/vendors/dimVendorItems.directive.js [hunk 1]
```diff
@@ -38,7 +38,7 @@
       '      <img class="vendor-icon" ng-src="{{::vendor.icon | bungieIcon}}" />',
       '      {{::vendor.name}}',
       '    </div>',
-      '    <timer class="vendor-timer" ng-if="vendor.nextRefreshDate[0] !== \'9\'" end-time="vendor.nextRefreshDate" max-time-unit="\'day\'" interval="1000">{{days}} day{{daysS}} {{hhours}}:{{mminutes}}:{{sseconds}}</timer>',
+      '    <timer class="vendor-timer" ng-if="vendor.nextRefreshDate[0] !== \'9\'" end-time="vendor.nextRefreshDate" max-time-unit="\'day\'" interval="1000">{{days}} <span ng-if="days === 1" translate="Vendors.Day"></span><span ng-if="days !== 1" translate="Vendors.Days"></span> {{hhours}}:{{mminutes}}:{{sseconds}}</timer>',
```
Comment (bhollis) [inline app/scripts/vendors/dimVendorItems.directive.js]: The translate plugin can handle plurals, I think. Let's use this as a test-bed for that.

I also think angular-timer supports localization more directly.

Comment (delphiactual) [inline app/scripts/vendors/dimVendorItems.directive.js]: angular-timer does but this is what happened when I tried adding dayUnits https://github.com/siddii/angular-timer/issues/222

**Comment by delphiactual** [comment]
@bhollis i18n has been adjusted to allow for pluralization and gender support... I added a new value onto store that is just the gender for ease of gender support

**Comment by bhollis** [comment]
Cool!

**Comment by kyleshay** [comment]
Awesome work & nice job figuring that out @delphiactual!

**Pull request merged**